sean-bloomington.md

```diff
@@ -0,0 +1,6 @@
+# Bloomington
+
+### Things to do
+- Miller Park
+- Bloomington Lake
+- Shooters :beer:
```

Comment:
@santakmishra Is this a bar with good beer selection, or more of a brew pub with in house brewed beer and food?